backend/sudokouBackend/src/main.rs

```diff
@@ -1 +1,13 @@
+mod board;
 
+use board::{SudokuGrid, SudokuError};
+
```

Comment:
The terms `board` and `grid` basically mean the same thing. Be consistent with only one.